### DISCUSSION TITLE
Why can't I use the Copilot student discount?

### DISCUSSION THREAD

**Discussion by Alohahaha-CC** [community/community#147186] [GitHub Education]
### Select Topic Area

Question

### GitHub Feature Area

Copilot

### Body

![屏幕截图 2024-12-16 115419](https://github.com/user-attachments/assets/20e4d5bd-7ef6-4a37-9b47-53b09d368d43)
![屏幕截图 2024-12-16 115446](https://github.com/user-attachments/assets/6c1192bc-afca-4545-a5b7-4ae0be8a59a8)
![屏幕截图 2024-12-16 115508](https://github.com/user-attachments/assets/faa1f3f4-715e-43ac-9f3f-bdddeefaf060)

**Comment by davevad93** [comment]
Hi @Alohahaha-CC ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Alohahaha-CC was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by Alohahaha-CC** [reply]
Thank you!So I just need to wait now,right?Should I send a request again?I have not finish the two-factor authentication.
Thanks a lot!

**Reply by davevad93** [reply]
You're welcome @Alohahaha-CC , you don't need to resubmit any application because your last one has been approved as shown in your screenshot, now you just have to wait, don't worry! 👍

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Alohahaha-CC ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you: **Education**. Good luck!
